app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from __future__ import annotations
+
+
+def calculate_team_total_rating(players: list) -> int:
```

Comment:
add full type hinting
list[Player]
The same in other functions. But with other type